mail_tracking/models/mail_message.py

```diff
@@ -94,11 +101,20 @@ def tracking_status(self):
             trackings = self.env['mail.tracking.email'].sudo().search([
                 ('mail_message_id', '=', message.id),
             ])
-            # Get Cc recipients
+            # String to List
             email_cc_list = email_split(message.email_cc)
+            email_to_list = email_split(message.email_to)
+            # Remove aliases recipients
+            self._drop_aliases(email_cc_list, db_aliases)
+            self._drop_aliases(email_to_list, db_aliases)
```

Comment:
```suggestion
```

Reply:
Can't avoid this :/... 
Without filter aliases here, we can print aliases when doesn't have a partner.